wayfindingsample/src/main/java/mappedin/com/wayfindingsample/VenuesCallback.java

```diff
@@ -16,9 +16,14 @@
 public class VenuesCallback implements MappedinCallback<List<Venue>> {
 
     private MappedIn mappedIn;
-    private LocationGenerator[] customerLocations;
     private MappedinCallback<Venue> getVenueCallback;
 
+    private LocationGenerator[] customerLocations;
```

Comment:
Lets use better names to explain what these are:

// Use this location generator if your data schema has only one location type with externalIds
// Most keys use this data schema
private LocationGenerator[]  genericLocations;

// Use this generator if your data schema uses multiple complex Location types 
// (tenant, amenity, elevator, escalatorStairs) you might have a different data schema
// A mappedin representative can help you set up any missing types
private LocationGenerator[] detailedLocations;